pipeline/aws-instances.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-REGIONS=( us-west-1 us-west-2 us-east-1 us-east-2 eu-central-1 eu-west-1 ap-northeast-1 ap-southeast-1 )
+REGIONS=( us-west-1 us-west-2 us-east-1 us-east-2 eu-central-1 eu-west-1 ap-northeast-1 ap-southeast-1 ap-southeast-2 )
```

Comment:
why adding ap-southeast-2?  Do we have all the setup in ap-southeast-2 region? like the key, vpc, security groups?